scripts/run-ecs-task.go

```diff
@@ -81,18 +80,26 @@ func main() {
 	handleError(err)
 
 	command := flag.String("command", "", "Command to be executed")
+	passCommandDirectly := flag.Bool("pass_command_directly", false, "true when run command without ls -lc")
```

Comment:
通常ハイフンかと！（kubectl とかでもそう）
```suggestion
	passCommandDirectly := flag.Bool("pass-command-directly", false, "true when run command without ls -lc")
```